subprojects/core/src/main/java/org/gradle/api/tasks/Copy.java

```diff
@@ -65,6 +66,7 @@
  * }
  * </pre>
  */
+@DisableCachingByDefault(because = "Is not CPU intensive")
```

Comment:
Yes, this task is mostly I/O. And we only cache CPU work, not I/O work. So I went with `Is not CPU intensive` for the tasks which do nothing which we can cache.

The same goes for `CreateStartScripts` for example, since it is also writing a file to disk without doing any CPU work.

Let's come up with something better together.